CMakeLists.txt

```diff
@@ -56,8 +56,8 @@ macro(build_benchmark)
   find_package(Patch REQUIRED)
 
   externalproject_add(benchmark-${GOOGLE_BENCHMARK_TARGET_VERSION}
-    URL "https://github.com/google/benchmark/archive/v${GOOGLE_BENCHMARK_TARGET_VERSION}/benchmark-${GOOGLE_BENCHMARK_TARGET_VERSION}.tar.gz"
-    URL_MD5 91d2d9a824cab82c67a80ccce5b93218
+    GIT_REPOSITORY "https://github.com/google/benchmark.git"
+    GIT_TAG 1302d2ce094a9753b0f81a81ea74c0fa71fae582
```

Comment:
Can't we just use the latest release v1.5.2?

Reply:
I will take a look. It must have been released recently it was not out last time I checked.

Reply:
yes we can I will change the commit